docs/upgrade.md

```diff
@@ -85,6 +85,15 @@ process, for example:
     dpkg -i matrix-synapse-py3_1.3.0+stretch1_amd64.deb
     ```
 
+# Upgrading to v1.43.0
+
+## The spaces summary APIs can now be handled by workers
+
+The [available worker applications documentation](https://matrix-org.github.io/synapse/latest/workers.html#available-worker-applications)
+has been updated to reflect that calls to the `/spaces`, `/hierarchy`, and
+`/summary` endpoints can now be routed to workers for both client API and
+federation requests.
```

Comment:
What's the intented action for server owners here---copy and and paste the new list of regexes into their config if they want to use workers?

Reply:
The intended action is to go look at the worker document and potentially update their config, yes.

We seem to have sometimes included upgrade notes about this in the past (we did for moving `createRoom`), but don't always seem to. It's possible this document is really only meant for things which *must* be done.

Reply:
Sounds good! Just wanted to check the thinking.